infra/horreum-dev-services/deployment/src/main/java/io/hyperfoil/tools/horreum/dev/services/deployment/HorreumDevServicesProcessor.java

```diff
@@ -176,8 +170,15 @@ public void startHorreumContainers(
                 throw new RuntimeException(t);
             }
 
-            devServicesResultBuildItemBuildProducer.produce(horreumKeycloakDevService.toBuildItem());
-            devServicesResultBuildItemBuildProducer.produce(horreumPostgresDevService.toBuildItem());
+            if (horreumPostgresDevService != null) {
+                devServicesResultBuildItemBuildProducer.produce(horreumPostgresDevService.toBuildItem());
+
+                // TODO: figure out how to run a SQL script to load dummy data into the database
```

Comment:
Can you remove the `TODO` comment and open a issue that we can track? thanks

Reply:
Added #1163 to track that feature